sources/packages/web/src/composables/useInvoiceBatchApprovalStatus.ts

```diff
@@ -0,0 +1,19 @@
+import { CASInvoiceBatchApprovalStatus, StatusChipTypes } from "@/types";
+
+export function useInvoiceBatchApprovalStatus() {
```

Comment:
Following the other ones, we can call the composable as `useInvoiceBatch` to be more generic.

Reply:
This one was following the supplier one but it also seems to be an exception. I will do the change since it is more generic.

Reply:
Decorator and file name adjusted.